napari_allencell_annotator/view/main_view.py

```diff
@@ -163,6 +164,29 @@ def _csv_json_import_selected_evt(self, file_path: Path):
 
             self.annots.start_viewing(use_annots)
 
+    def _process_points_annotations(self, annotations: list[any]) -> list[any]:
+        """
+        Convert string representations of point annotations to lists of tuples
+
+        Parameters
+        ----------
+        annotations: list[any]
+            The list of annotations read from the CSV
+
+        Returns
+        -------
+        list[any]:
+            The same list of annotations with points annotations as lists of tuples
+        """
+        # if that image has been annotated
+        if len(annotations) > 0:
+            for annotation_index, key in enumerate(self._annotator_model.get_annotation_keys().values()):
+                # if the annotation is point annotation and has been annotated, convert string to list of tuples.
+                if key.get_type() == "point" and annotations[annotation_index] != "":
+                    annotations[annotation_index] = ast.literal_eval(annotations[annotation_index])
```

Comment:
you can do this with just `eval`

Reply:
changed in https://github.com/aics-int/napari-allencell-annotator/pull/96/commits/c919e7dafd5aabb574c0e9b4a02957c75f4ad4ec